### DISCUSSION TITLE
Duplicate services

### DISCUSSION THREAD

**Discussion by allanjthorne** [maxileith/homebridge-appletv-enhanced#419] [Q&A] [answered]
I am using homebridge with this appletv-enhanced plugin and the nodered plugin. If I enable more than one Apple TV device (I have 2) then I get duplicate entries for source inputs (apps on each device) eg. Disney+, Spotify, AppleTV. Is there a way to change the device input source name for each device to remove the duplicates?

**Comment by maxileith** [comment]
That should definetly not happen. Could you please file a bug report in the issues section?

**Reply by allanjthorne** [reply, marked as answer]
Thank you for your response. I have put a workaround in place. I have BlackListed one of the Apple TV's that I do not need to control via Homebridge / NodeRed. I do not have any log files as the problem happened a while ago and I do not wish to change my current configuration. The problem seems to be with the NodeRed plugin, not your plugin.
